tests/test_imod_coupler/test_ribamod.py

```diff
@@ -148,23 +148,23 @@ def test_ribamod_backwater(
     riv0 = results.budgets["riv-1"].isel(time=-2).compute()
 
     # Get the last flow between the edges
-    final_flow = results.flow_df[results.flow_df["time"] == "2020-12-31"]
+    volume_balance = results.basin_df[results.basin_df["time"] == "2020-12-31"]
```

Comment:
again?

Reply:
This is the corresponding volume balance, the other was the level